astropy/constants/astropyconst20.py

```diff
@@ -12,6 +15,34 @@
 
 _utils._set_c(codata, iaudata, find_current_module())
 
+# Overwrite the following for consistency.
+# https://github.com/astropy/astropy/issues/8920
+with warnings.catch_warnings():
+    warnings.simplefilter('ignore')  # Ignore redefinition warnings
```

Comment:
```suggestion
    warnings.filterwarnings('ignore', 'Constant .*already has a definition')
```
This is used in `__init__.py` so that we don't filter out all warnings.